Adds information about custom .sqlfluff and .sqlfluffignore files (#3928)
fishtown-analytics/docs.getdbt.com: website/docs/guides/best-practices/how-we-style/2-how-we-style-our-sql.md

## Basics

- ☁️ Use [SQLFluff](https://sqlfluff.com/) to maintain these style rules automatically.
  - Customize `.sqlfluff` configuration files to your needs.
  - Refer to our [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we us in our own projects. 

>   - We also support ignoring files and folders using a standard `.sqlfluffignore` file. Reference these docs for the syntax: [.sqlfluffignore syntax docs](https://docs.sqlfluff.com/en/stable/configuration.html#id2)

Reviewer: ```suggestion
  - Exclude files and directories by using a standard `.sqlfluffignore` file. Learn more about the syntax in the [.sqlfluffignore syntax docs](https://docs.sqlfluff.com/en/stable/configuration.html#id2).
```

## Basics

- ☁️ Use [SQLFluff](https://sqlfluff.com/) to maintain these style rules automatically.

>   - Custom `.sqlfluff` configuration files are supported.

Reviewer: ```suggestion
  - Customize `.sqlfluff` configuration files to your needs.
```

>   - Reference this [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we use in our own projects.

Reviewer: ```suggestion
  - Refer to our [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we us in our own projects. 

```
